src/curves/jubjub.rs

```diff
@@ -1,10 +1,13 @@
 use std::str::FromStr;
 
-use crate::{Bls12_381, CyclicGroup, EdwardsCurve, EdwardsExpression, EdwardsPoint, Element, Group};
+use crate::{Bls12_381, CyclicGenerator, EdwardsCurve, EdwardsGroup, EdwardsExpression, EdwardsPoint, Element, Group};
 
-pub struct JubJubPrimeSubgroup {}
+pub struct JubJub {}
 
-impl EdwardsCurve<Bls12_381> for JubJubPrimeSubgroup {
+impl JubJub {
```

Comment:
Nit: Are there any issues with an empty `impl`? I oddly haven't seen this before so I don't know if there is any potential for inefficiency, etc.